src/sentry/api/endpoints/source_map_debug.py

```diff
@@ -90,8 +94,25 @@ def get(self, request: Request, project: Project, event_id: str) -> Response:
         frame, filename, abs_path = self._get_frame_filename_and_path(exception, frame_idx)
 
         if frame.context_line:
+            # the frame is already source mapped
             return self._create_response()
 
+        sdk_info = event.data.get("sdk")
+        if (
+            sdk_info
+            and sdk_info["version"] < JS_VERSION_FOR_DEBUG_ID
```

Comment:
I don't know Python well enough to tell if this comparison will always work. Will it? 👀

Reply:
I think so...what case won't it work? It's just a string compare. Example edge case:

![Screen Shot 2023-05-15 at 9 29 44 AM](https://github.com/getsentry/sentry/assets/8533851/7ea331f2-6764-4ae2-879d-a1226510f4b1)

Reply:
@scefali can you confirm that python won't run into the same issue? is there a better approach we can use here to avoid future problems?

Reply:
We have a version class in our python code base that overloads the `>` and `<` operators to work with semver. As long as we use this class for `sdk_info['version']` here we're good.

Reply:
@priscilawebdev this is Pythons output, so it will 🥲

Reply:
oh yes... I read Node 10 but the code is in python...it makes sense